cms/models_test.py

```diff
@@ -1785,3 +1790,25 @@ def test_program_page_price_is_updated(superuser_client):
     resp = superuser_client.post(path, data_to_post)
     assert resp.status_code == 302
     assert program.current_price == 999
+
+
+def test_course_run_certificate_get(user_client):
+    """Test that course run certificate get is successful for a valid UUID and raises 404 for invalid UUID"""
+    certificate = CourseRunCertificateFactory.create()
+    resp = user_client.get(f"/certificate/{certificate.uuid}/")
+    assert resp.status_code == 200
+
+    invalid_uuid = str(certificate.uuid)[0:-12]
+    resp = user_client.get(f"/certificate/{invalid_uuid}/")
+    assert resp.status_code == 404
+
+
+def test_program_certificate_get(user_client):
+    """Test that program certificate get is successful for a valid UUID and raises 404 for invalid UUID"""
+    certificate = ProgramCertificateFactory.create()
+    resp = user_client.get(f"/certificate/program/{certificate.uuid}/")
+    assert resp.status_code == 200
+
+    invalid_uuid = str(certificate.uuid)[0:-12]
+    resp = user_client.get(f"/certificate/program/{invalid_uuid}/")
+    assert resp.status_code == 404

```

Comment:
nit: We can refactor this in this way:

1. A test only to check the valid certificate for both course and program certificate
2. A parametrized test to check invalid certificates for both course & program certificate

We should check on multiple invalid UUIDs. Ideally making these tests parameterized for invalid  e.g. 

1. Short string
2. Empty String
3. String longer than the length of UUID

Reply:
There were no existing tests so I just tried to put in some basic ones quickly but sure, I will add some more as you suggested.